src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
Hey there, great work on the `calculateMenAverageAge` function. However, I noticed a couple of areas where you could refine your code.

In your current implementation, you're using two separate `filter` operations based on the `century` parameter. This is unnecessary as you can combine these into a single `filter` operation and check for the `century` within the same operation.

```suggestion
 const men = people.filter(person => person.sex === 'm' && (!century || Math.ceil(person.died / 100) === century));
```